src/engine/zsm.cpp

```diff
@@ -66,6 +66,8 @@ void DivZSM::init(unsigned int rate) {
   // Channel masks
   ymMask=0;
   psgMask=0;
+  // Optimize writes
+  optimize=false;
```

Comment:
Consistency; should be true if it is on by default.

Reply:
fair point :)